### DISCUSSION TITLE
gmap icons to osmand favorites converter (kmz -> gpx)

### DISCUSSION THREAD

**Discussion by mariush444** [osmandapp/OsmAnd#13190] [Ideas]
I think about converter of kmz file to gpx based on https://www.mail-archive.com/osmand@googlegroups.com/msg07817.html
I try to make mapping https://github.com/mariush444/gmapIcons2osmand/blob/main/icons-gmap-osmand.pdf
but I wonder if osmand available icons for favorites are OK.

1. Shouldn't favorites icons be more general?
![obraz](https://user-images.githubusercontent.com/66887280/142009199-2b569d5d-f35a-465b-8b5c-87c0fa8bf100.png)
How often can TajMahal, Eiffel or Colosseum be used?
Shouldn't be palace, tower, ruins ... ?
I understand that I can use Brandenburg gate as antic architecture but same shapes seems to be such unambiguous.

2. Do you plan to allow access to icons stored in https://github.com/osmandapp/OsmAnd-resources/blob/master/rendering_styles/style-icons/poi-icons-png/ by user in favorites screen?
These icons are available when I edit gpx directly but it is impossible to use them via UI.

**Comment by vshcherb** [comment]
We plan to add most of the icons but we need to categorize them first. Also we plan to store icon from POI itself if Favorite was created on top POI
